snap/snapcraft.yaml

```diff
@@ -133,6 +153,38 @@ parts:
     stage-packages:
       - inotify-tools
 
+  cuda-samples:
+    source: https://github.com/NVIDIA/cuda-samples
+    source-tag: v11.5
+    source-type: git
```

Comment:
Should we try and just build latest? It's 12.1 by now - and I imagine we'll be falling behind quickly if we stick to a version. I suppose could subscribe to release notifications there…

```suggestion
```

Reply:
This was the highest version at the time, compatible with the -server version of drivers we used at the time, compatible with focal & jammy. I can check if 12.x series are usable for us now, but they weren't compatible before when we were on older generation of driver =/

Reply:
Your call.